### ISSUE TITLE
Default System Profile behaviour

### ISSUE THREAD

**Issue by kgoderis** [eclipse/smarthome#4669]
With reference to https://github.com/eclipse/smarthome/pull/4516, I wonder if it would not more logical to revert back the change with respect to handleUpdate(). That is, the Profiles introduced up to now do actually enable a behaviour. Therefore, it would be more logical to have "no behaviour" by default, and enable the current default behaviour with an explicitly defined profile. 

This would also be more logical from a user point of view, whereby a new wanted behaviour has to be added explicitly to the Channel definition.

**Comment by maggu2810** [comment]
Hi @kgoderis, I am not sure if I understand your suggestion?
Isn't the current default profile that handle command and handle update are delegated to the handler?
What is your suggestion a default profile should look like?

**Comment by kaikreuzer** [comment]
@kgoderis Isn't this a duplicate of the discussion that is supposed to be done in https://github.com/eclipse/smarthome/issues/4525?

**Comment by kgoderis** [comment]
@kaikreuzer partly, in the sense that we discuss the handleUpdate() function itself in #4525

@maggu2810 Before #4516 there was no flow of Command and States between Channels connected to the same Item, e.g. default behaviour was "do nothing". All the profiles introduced up to now "add" a behaviour, or enable a behaviour. They are "additive" to the link.  With #4516 the default is behaviour is "do something", with all the defined profiles interpreted as "replace default behaviour by another behaviour". I find it more logical to have no default behaviour, and all the Profiles to be "additive". Just thinking forward to the day that there will be a use case to have more than one profile on an ItemChannelLink..... 

The changed default behaviour actually breaks things very badly in my installation, and more specifically with the KNX binding, which in itself is under discussion / not progressing at the same speed, but which is a primary future user of Profiles just in order to prevent loops of Telegrams on the KNX bus. I would hope that this change of default behaviour can be reverted at least until there is a clear view how things work out for the KNX binding. KNX is maybe not a primary concern for ESH, but it is the core technology in a few installations, notably of contributors like @kaikreuzer ;-)

**Comment by kaikreuzer** [comment]
> With #4516 the default is behaviour is "do something"

No, this is not correct. The default profile is completely agnostic and simply passes commands and state updates between the framework and the binding as it used to do before.

Note that it specifically states "Restore backward-compatibility by making the default master profile forward handleUpdate() calls to ThingHandlers again". So it didn't introduce any changed behavior, it simply fixed a regression that was introduced shortly before.

> Just thinking forward to the day that there will be a use case to have more than one profile on an ItemChannelLink.....

There is definitely no plan to do so - there should be only a single profile per link, everything else might be contradicting. Instead, you can link the same item to different channels using different profiles.

In short, I still think this is a duplicate of https://github.com/eclipse/smarthome/issues/4525 and should be closed.

**Comment by kaikreuzer** [comment]
>  I still think this is a duplicate of #4525 and should be closed.

No complaints, so I close it :-)
